src/base58.cpp

```diff
@@ -4,16 +4,8 @@
 
 #include "base58.h"
 
-#include "hash.h"
-#include "uint256.h"
 
-#include <assert.h>
-#include <stdint.h>
 #include <string.h>
-#include <vector>
```

Comment:
`std::vector` is used in this file, so the include is necessary. I imagine this is true of many other removed includes...